adoc/i-Introduction/SA.adoc

```diff
@@ -1,40 +1,42 @@
 
 image::SLES4SAPBenefits.png[Benefits of SLES-for-SAP Applications,scaledwidth="80%"]
 
-*Simplify*::      the deployment of an SAP Landscape in Azure for dev, test, and production.
+*Simplify*::      the deployment of an SAP Landscape in The Public Cloud for dev, test, and production.
 
-*Modernize*::     customer environments by taking advantage of the power of the public cloud
+*Modernize*::     customer environments by taking advantage of the benefits of the public cloud
 
 *Accelerate*::    customer migrations to the cloud.
 
 
-Starting from the idea of simplifying and modernizing SAP HANA and SAP NetWeaver deployments, SUSE worked on rewriting the deployment wizards we built.
+Building the infrastructure to run SAP Applications can be complex and demands significant effort, especially if a manual deployment method is used. When delivering muliple envionments, for mulitple engagements, reproducing the deployment can be tedious and error-prone. 
+
+One additional challenge is ensuring the infrastructure is highly available, as this will add more complexity and steps to the deployment.
 
-Building the infrastructure to running SAP Applications can get quite complex and demands big effort if they get deployed manually, and in addition reproducing it, can be tedious and error-prone. 
+When deploying and managing a large number of systems in an SAP Landscape, there is often a secondary need, getting an overview of what is happening in he environemt after the install is complete.
 
-An additional challenge is to make the infrastructure high available, as this will add more complexity and tasks.
+To help with this, SUSE has added the ability gain insight into your SAP Landscape with a comprehensive monitoring solution, providing dashboards, realtime and historic views and active alerts and reporting.
 
-Such a number of systems in SAP Landscapes have an further need - getting an overview whats going on after the install is done. Therefore we added the possibility to allow insights into your SAP Landscape with comprehensive dashboards, realtime and historic views and active alerts and reporting.
 
-The major motivation was to improve, simplify and unify the installation of SAP Landscape on SUSE Linux Enterprise Server for SAP Applications and clearly standardize deployments and allow customers to use one level of tooling in various ways – from a Command Line interface, through some GUI driven process and SUSE Manager or other automation frameworks.
+The key goals are:
+- improve, simplify and unify the installation of SAP Landscape on SUSE Linux Enterprise Server for SAP Applications 
+- clearly standardize deployments and allow customers to use one set of tooling for multiple use cases such as Command Line interface, a GUI driven process, SUSE Manager or other automation framework.
```

Comment:
will take back the original sentence, as your change did change the meaning complete. 

I want to say: use one tool in various ways
you say        : use  one tool for multiple use cases

I think this is really different